cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js

```diff
@@ -66,6 +66,8 @@ const cpu_dir = Gio.file_new_for_path(cpu_path);
 let height = 22;
 const DEC_WHITE = 16777215;
 
+const formatNum = new Intl.NumberFormat(undefined, { maximumFractionDigits: 2, minimumFractionDigits: 2 }).format;
```

Comment:
```suggestion

```

Reply:
Isn't better to format numbers using locale aware framework? For example, some locales use comma for decimal separators.
(Just checked, Cinnamon applets runs with the locale taken from the system)

Reply:
actually, I'd also change all other number presentations to use this formatter (will keep it for next PR)